neofs/src/main/java/io/neow3j/neofs/sdk/NeoFSClient.java

```diff
@@ -77,6 +88,14 @@ private static String createClient(NeoFSLib lib, Account account, String neofsEn
         return new String(getResponseBytes(response));
     }
 
+    private static String getPrivateKeyForNativeLib(ECKeyPair ecKeyPair) {
+        return Numeric.toHexStringNoPrefix(ecKeyPair.getPrivateKey().getBytes());
+    }
+
+    public static String getPrivateKeyForNativeLib(Account account) {
+        return getPrivateKeyForNativeLib(account.getECKeyPair());
+    }
+
     //region client
 
     // Todo: Delete a client
```

Comment:
the delete client part will not be done? Like, disposal of a client?

If not, can you create an issue about that?

Reply:
Yes, I want to include that. I'll open an issue about it.

Reply:
This has been done in [09d3cb9](https://github.com/neow3j/neow3j/pull/908/commits/09d3cb97a954a7f3c16a7f0dea70e1e818babc85).